packages/html/src/column-menu/column-menu-item-content.tsx

```diff
@@ -0,0 +1,17 @@
+import { classNames } from '../misc';
+
+const className = `k-columnmenu-item-content`;
+
+export const ColumnMenuItemContent = (
+    props: React.HTMLAttributes<HTMLDivElement>
+) => (
+    <div
+        className={classNames(
```

Comment:
Add `{...props}` or things like `style` and other props defined on `<HTMLDivElement>` won't apply if you ever need them.